internal/consensus/state_test.go

```diff
@@ -580,7 +584,7 @@ func TestStateLock_NoPOL(t *testing.T) {
 
 	ensureNewTimeout(t, timeoutWaitCh, height, round, cs1.config.Precommit(round).Nanoseconds())
 
-	cs2, _ := makeState(ctx, t, config, logger, 2) // needed so generated block is different than locked block
+	cs2, _ := makeState(ctx, t, config, logger, makeStateArgs{validators: 2}) // needed so generated block is different than locked block
```

Comment:
The comment was pre-existing, but I'm not sure what "needed" refers to here. Is it the definition of cs2, or the request for 2 validators, or…?

Reply:
cs2 has to be created to generate a new block, otherwise the locked block will be used.

Reply:
updated, PTAL